### PR TITLE
Our vision/story

### PR DESCRIPTION
**Pull request by svanen99**

### CODE REVIEW

**Review by Legendarius-K — CHANGES_REQUESTED** [review]
Looks awesome! A thinner font on the text then done!

**Review by Legendarius-K — APPROVED** [review]
CR approved! @svanen99

File: src/components/OurVisionStory/index.tsx
Comment (timdeckel) [inline src/components/OurVisionStory/index.tsx:1]: there is some code here that is commented out and looks like legacy code. some clean up could be nice.

**Review by timdeckel — APPROVED** [review]
code is tested and checked. fix the commented code please. other that it looks good!

**Pull request merged**